tests/integration/pilot/common/routing.go

```diff
@@ -2322,7 +2322,7 @@ spec:
       consistentHash:
         httpCookie:
           name: session-cookie
-          ttl: 3600s
+          ttl: 0s
```

Comment:
confused this is "with TTL" but its 0s?

Reply:
was just trying to make it a session cookie and see if it matters - with ttl just means letting envoy generate the cookie instead of client passing. It does not seem to make much difference. I can put it back